lib/controller.dart

```diff
@@ -61,6 +64,18 @@ class ToggleOptionEnabledData extends Data {
   ToggleOptionEnabledData(this.isEnabled);
 }
 
+class ToggleActiveFilterData extends Data {
+  String key;
+  bool isEnabled;
```

Comment:
as in the other PR, prefer to avoid `is*` construction if the name is already fairly suggestive of having a bool value

Reply:
Yep will issue a fix after i merge it in - so I can replace the symbol names without conflicts